docs/pages/workflow/glossary-of-terms.md

```diff
@@ -111,14 +338,56 @@ We use the word "slug" in [app.json](#appjson) to refer to the name to use for y
 
 [Snack](https://snack.expo.dev/) is an in-browser development environment where you can build Expo [experiences](#experience) without installing any tools on your phone or computer.
 
+### Software Mansion
+
+A development agency in Krakow, Poland. Maintainers of `react-native-gesture-handler`, `react-native-screens`, and `react-native-reanimated`. The platform team at Expo is compromised of a number of contractors from Software Mansion. All of Software Mansion's core React Native libraries are supported in [Expo Go](#expo-go).
+
 ### Standalone app
 
 An application binary that can be submitted to the iOS App Store or Android Play Store. [Read more in "Building Standalone Apps"](../distribution/building-standalone-apps.md).
 
+### Store Config
+
+The `store.config.json` file used to configure [EAS Metadata](#eas-metadata). This file can be generated from an existing App Store entry using `eas metadata:pull`.
+
+### Sweet API
+
+The Swift and Kotlin API for writing React Native modules. This API is provided by the library `expo-modules-core` which is shipped with the `expo` package. [Learn more](/modules/module-api/).
+
+### TypeScript
+
+TypeScript is a strongly typed programming language that builds on JavaScript, giving you better tooling at any scale. The Expo SDK is written in TypeScript, we highly recommend using it. [Learn more](/guides/typescript/).
+
+### Updates
+
+Traditionally, apps for iOS and Android are updated by submitting an updated binary to the App and Play stores. Updates allow you to push an update to your app without the overhead of submitting a new release to the stores. [Read more in "Publishing"](publishing.md).
+
+### VS Code Expo
+
+The VS Code extension for improving the developer experience of working with Expo config files. This extension provides autocomplete and intellisense for the [Expo Config](#expo-config), [Store Config](#store-config), [Expo Module Config](#expo-module-config), and [EAS Config](#eas-config). [Learn more](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo).
```

Comment:
```suggestion
The VSCode extension for improving the developer experience of working with Expo config files. This extension provides autocomplete and intellisense for the [Expo Config](#expo-config), [Store Config](#store-config), [Expo Module Config](#expo-module-config), and [EAS Config](#eas-config). [Learn more](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo).
```

Let's stick the branding name here for VSCode.

Reply:
VS Code is a preferred (correct?) abbreviation IIRC.

<img width="1243" alt="Screenshot 2022-08-11 at 13 01 40" src="https://user-images.githubusercontent.com/719641/184119575-d38ebce6-88d7-4dd8-8ec9-605565267e6b.png">

Reply:
Oh, interesting. It's kinda confusing how they are branding themselves 😅

You are right! My bad! I thought, they were branding themselves as `VSCode` but from their own docs it seems its `VS Code`. I'll remove this suggestion and update our writing style guide. Thanks for having this conversation on it :)